src/map/packets/char_health.h

```diff
@@ -34,14 +34,13 @@
 *																		*
 ************************************************************************/
 
-class CCharEntity;
+class CBattleEntity;
 class CFellowEntity;  // NPCFELLOW
 class CCharHealthPacket : public CBasicPacket
 {
 public:
 
-	CCharHealthPacket(CCharEntity* PChar);
-    CCharHealthPacket(CFellowEntity* PFellow);  // NPCFELLOW
+	CCharHealthPacket(CBattleEntity* PChar);
```

Comment:
Should this be:
```   
CCharHealthPacket(CBattleEntity* PEntity);
```
to match the one in char_health.cpp?

Reply:
you can change it if you want, it's just the name so it doesn't particularly matter but it's probably better to change it